website/src/examples/home-demo.js

```diff
@@ -0,0 +1,135 @@
+import {makeExample} from '../components';
+
+import React, {Component, useEffect, useMemo, useRef} from 'react';
+import {Map} from 'react-map-gl/maplibre';
+import DeckGL from '@deck.gl/react';
+import {H3HexagonLayer} from '@deck.gl/geo-layers';
+import {PathStyleExtension} from '@deck.gl/extensions';
+import {getRes0Cells, isValidCell, uncompactCells} from 'h3-js';
+
+const INITIAL_VIEW_STATE = {
+  longitude: -74,
+  latitude: 40.72,
+  zoom: 13,
+  pitch: 45,
+  bearing: 0
+};
+
+const MAP_STYLE = 'https://basemaps.cartocdn.com/gl/positron-gl-style/style.json';
+
+export function App({
+  hex = undefined,
+  initialViewState = INITIAL_VIEW_STATE,
+  mapStyle = MAP_STYLE,
+}) {
+  const deckRef = useRef();
+  const res0Cells = useMemo(() => getRes0Cells().map((hex) => ({hex})), []);
+  const res1Cells = useMemo(() => uncompactCells(getRes0Cells(), 1).map((hex) => ({hex})), []);
+  const res2Cells = useMemo(() => uncompactCells(getRes0Cells(), 2).map((hex) => ({hex})), []);
+  const userValidHex = useMemo(() => isValidCell(hex), [hex]);
+  useEffect(() => {
+    if (userValidHex && deckRef.current) {
+      //TODO: zoom to bounds here...
+    }
+  }, [hex, userValidHex]);
+
+  const layers = userValidHex? [new H3HexagonLayer({
+    id: "userhex",
+    data: [{hex}],
+    getHexagon: (d) => d.hex,
+    extruded: false,
+    filled: false,
+    stroked: true,
+    getLineColor: [0, 0, 0],
+    getLineWidth: 3,
+    lineWidthMinPixels: 3,
+    highPrecision: true,
+  })] : [
+    new H3HexagonLayer({
```

Comment:
Are you intending to implement support for finer resolutions (viewport-only)? Or is the idea to just show the first 3 resolutions? If it's the latter, this might actually be simpler and more performant just with Mapbox/Maplibre styles, rather than Deck...